test/test_change_stream.py

```diff
@@ -158,10 +162,14 @@ def test_try_next(self):
         with self.change_stream(max_await_time_ms=250) as stream:
             self.assertIsNone(stream.try_next())  # No changes initially.
             coll.insert_one({})  # Generate a change.
+
```

Comment:
Unneeded added space here.

Reply:
Yeah...I'll try again for sure, but i tried to delete it and when i ran pre-commit it was added again??? not sure why though

Reply:
Because the async file overwrites the sync one

Reply:
ohh, I think it is because ruff is just doing that because the file has changed

Reply:
yeah so I found the space in the async file, deleted it, and then it got added back during pre-commit

Reply:
Ah in that case never mind, if the formatter adds an extra space that's fine.

Reply:
perfect, because I just gave up on trying to convince it otherwise